gnomad_qc/v3/create_release/prepare_vcf_data_release.py

```diff
@@ -381,6 +387,10 @@ def make_info_expr(
 
     vcf_info_dict["primate_ai_score"] = t["primate_ai"]["primate_ai_score"]
 
+    # Add VRS annotations to info dict
+    for field in list(VRS_FIELDS_DICT.keys()):
+        vcf_info_dict[field] = t["release_ht_info"]["vrs"][f"{field}"]
```

Comment:
looks right! seems like the same structure as the rest of the vcf_info_dict additions